tasks/Putting food into categories.md

```diff
@@ -0,0 +1,52 @@
+> 
+> Supporting 3: http://users.ece.utexas.edu/~perry/education/382v-s08/papers/mcgrath84.pdf
+
+# Stimuli
+
+## Materials for alternative versions of the experiment 
+> If the task has multiple versions based on changing stimuli (e.g. the set of eyes in RME), if possible, provide a complete set, or a specification of the rules for generating that set. For example, in the RME task, provide all of the eye images, and a table of terms.  
+> In the case that the complete set is not possible, provide enough unique examples that the reference version of the task experiment could be completed. If there are various levels of the task, e.g., in Sudoku, include a similarly sized sample of each level.
+
+# Procedure
+## Steps
+>Step one: Twenty-two three person groups will perform this task simultaneously in separate rooms.
+>
+>Step two: The open group condition will be notified that their group will be reconstituted during the second part of the task. This should not be told to the closed group.
+>
+>Step three: The groups will then perform task one for 10 minutes.
+>
+>Step four: Then, the open group condition would be told that one of the group members would join another group, and another individual who had completed part one in another group would their group. This is done randomly.
+>
+>Step five: Before beginning part two, all of the groups should spend 5 minutes talking about their group task. Then they will perform part two for 10 minutes.
+
+## Roles 
+> The open group condition is randomly selected, but made aware that before part two of their task, a member of their group will leave and be replaced by a person who had completed part one in a different group. That member will then join another group.
+> The closed group condition will not be made aware of this, and will complete part one, spend 5 minutes discussing the task as the open group does, and then complete part two.
+
+## Instructions
+> You will be asked to join a three person group and move into private rooms. Select a member of the group to record the group's ideas.
+> 
+> Part 1: View the list of 12 fruit items and generate as many criteria to classify these items into subgroups in 10 minutes.
+> 
+> Spend 5 minutes discussing the group task
+> 
+> Part 2: View the list of 12 vegetable items and generate sorting critera in 10 minutes.
+
+# Criteria
+## Performance calculation
+> Creativity was measured by fluency and flexibility.
+> Fluency was calculated by two raters who independently coded each transcript of ideas for the total number of non-repetitive ideas.
+> Flexibility was calculated by two raters independently as well, but they classified ideas into mutually exclusive idea categories and counted the total number of non-repetitive idea categories used by each group (ex. physical characteristics, seasonality).
+
```

Comment:
Should we add that raters were blind to the experimental conditions? Also should they be added to the "roles" section? I could go either way since it doesn't seem like they were actually at the experiment

Reply:
I think that could be helpful thanks!